tests/unit/misc/test_keyhints.py

```diff
@@ -0,0 +1,113 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2016 Ryan Roden-Corrent (rcorre) <ryan@rcorre.net>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test the keyhint widget."""
+
+from collections import OrderedDict
+import pytest
+
+from qutebrowser.misc.keyhintwidget import KeyHintView
+
+
+def expected_text(*args):
+    """Helper to format text we expect the KeyHintView to generate
+
+    Args:
+        args: One tuple for each row in the expected output.
+              Tuples are of the form: (prefix, color, suffix, command).
+    """
+    text = '<table>'
+    for group in args:
+        print("group = {}".format(group))
+        text += ("<tr>"
+                 "<td>{}</td>"
+                 "<td style='color: {}'>{}</td>"
+                 "<td style='padding-left: 2ex'>{}</td>"
+                 "</tr>").format(*group)
+
+    return text + '</table>'
+
+
+@pytest.fixture
+def keyhint(qtbot, config_stub, key_config_stub):
+    """Fixture to initialize a KeyHintView."""
+    config_stub.data = {
+        'colors': {
+            'keyhint.fg': 'white',
+            'keyhint.fg.suffix': 'yellow',
+            'keyhint.bg': 'black'
+        },
+        'fonts': {'keyhint': 'Comic Sans'},
+        'ui': {'show-keyhints': True},
+    }
+    keyhint = KeyHintView(0, None)
+    #qtbot.add_widget(keyhint)
```

Comment:
I copied from another test and wanted to see what would break if I commented it
out :fearful: 

Nothing obvious lew up so I guess I forgot about it.